source/computer/wsl-setup.rst

```diff
@@ -1,18 +1,143 @@
 WSL 配置指南
 ============
 
+:本节贡献者: |赵志远|\（作者）、
+             |姚家园|\（审稿）、
+             |田冬冬|\（审稿）
+:最近更新日期: 2021-01-18
+:预计花费时间: 120 分钟
+
+简介
+-----
+
+WSL 有 WSL1 和 WSL2 两个发行版本，二者底层原理不同。大多数情况下，建议使用 WSL2，
+因为它提供更快的性能和 100% 的系统调用兼容性。但是，当项目需要经常从 Linux 上访问存储
+在 Windows 系统上的文件，或利用 Windows 软件对 Linux 上的文件进行处理时，使用 WSL1 具有
+更快实现的性能。
+
+官方目前没有弃用 WSL1 的计划，并且支持将任何一个已经安装的 Linux 发行版更改为 WSL1 或者 WSL2。
+
+安装 WSL
+---------------
+
+WSL 的安装方式可以参考官方文档：
+
+- 中文指南：https://docs.microsoft.com/zh-cn/windows/wsl/install-win10
+- 英文指南：https://docs.microsoft.com/en-us/windows/wsl/install-win10
+
+官方文档对 WSL 安装配置中可能遇到的常见问题都给出了详细的解决方案，请认真阅读。
+
+WSL 可以安装不同的 Linux 发行版，但目前官方并未提供 Fedora 发行版的安装。
+推荐使用 WSL 安装 Ubuntu 20.04 LTS 或 Ubuntu 20.10。
+
+.. note::
+
+   开启 WSL 后，Linux 发行版的默认安装位置是 C 盘。安装多个发行版会占用 C 盘的大量空间。
+   可以先将一个新安装的 Linux 发行版导出备份，然后将其还原到其它盘，最后删除 C 盘上的发行版。
+   这样可以保证 C 盘有足够的空间。
+
+配置 Linux
+-----------
+
+安装 WSL 之后，还需要对 Linux 发行版进行一定配置。
+
+如果你安装了 Ubuntu，可以参考 《\ :doc:`/computer/ubuntu-setup` 》进行配置。
```

Comment:
Perhaps I mentioned before, we can move this section to the end.

Reply:
I already move `X Server` to the end.

Reply:
I mean the "setting up Linux" section.

Reply:
I mean `X Server` section may be the end, and `setting up Linux` may be the one before last?

Reply:
Done~